src/main/scala/io/appthreat/atom/Atom.scala

```diff
@@ -169,8 +172,10 @@ object Atom {
             JSConfig(disableDummyTypes = true).withInputPath(config.inputPath).withOutputPath(config.outputAtomFile)
           )
           .map { cpg =>
-            new OssDataFlow(new OssDataFlowOptions(maxNumberOfDefinitions = config.maxNumDef))
-              .run(new LayerCreatorContext(cpg))
+            if (config.withDataDeps) {
+              new OssDataFlow(new OssDataFlowOptions(maxNumberOfDefinitions = config.maxNumDef))
+                .run(new LayerCreatorContext(cpg))
+            }
             new JavaScriptInheritanceNamePass(cpg).createAndApply()
```

Comment:
Would such passes work without the dataflow layer?

Reply:
Yes, much of the post-processing doesn't require it. The only thing that this is necessary for is the data flow slicing. I suppose I should then set `config.withDataDeps = true` if `dataflow` mode is on.